Bot.py

```diff
@@ -58,12 +58,9 @@ def __init__(self) -> None:
             self._api = tweepy.API(auth)
             # Test the authentication. This will gracefully fail if the keys aren't present.
             try:
-                self._api.rate_limit_status()
-            except TweepError as e:
-                if e.api_code == 215:
-                    log.error("No keys or bad keys")
-                else:
-                    log.error("Other API error: {}".format(e))
+                self._api.verify_credentials()
+            except tweepy_unauthorized as e:
+                log.error("No keys or bad keys")
```

Comment:
Might be good to log the exception here too:
```suggestion
                log.error(f"No keys or bad keys: {e}")
```